src/app/containers/app/app.component.spec.ts

```diff
@@ -1,20 +1,20 @@
-import { TestBed, async } from '@angular/core/testing';
-import { AppComponent } from './app.component';
-import { combineReducers, StoreModule, Store } from '@ngrx/store';
-import {Logout, reducers} from 'src/app/store';
-import { HeaderComponent } from '../header/header.component';
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, TestBed } from '@angular/core/testing';
+import { combineReducers, Store, StoreModule } from '@ngrx/store';
 import { cold } from 'jasmine-marbles';
+import {Logout, reducers} from 'src/app/store';
```

Comment:
@SivaprakasamSivaprakasam - change this import to use a relative path instead of one starting with `src/`